### ISSUE TITLE
Can't see content in my library tab and songs in any playlists.

### ISSUE THREAD

**Issue by Teeworlds4** [ciderapp/Cider-2#713]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ciderapp/Cider-2/issues) for a bug report that matches the one I want to file, without success.
- [X] I have checked that the correct behavior occurs on [Apple Music Web](https://music.apple.com) (if applicable).
- [X] I have verified that my issue is with a [supported version](https://github.com/ciderapp/Cider-2/blob/main/SECURITY.md) and that it is with the desktop app (Support for other areas can be found in our Discord / on the appropriate repo).

### Download Source

Itch

### Operating System

Windows 11

### Commit Hash

786fea9c1d7a026cef43ff945334ff66b9ec694e

### Connect Instance ID

not-logged-in

### Build Timestamp

16.08.2024, 06:33:20

### Describe the Bug

So basically i cannot see any songs, albums and artists in my library tab expect in the recently added tab. And also i can't see content of any playlist after the update.

### Steps to Reproduce

1. Delete %appdata%/C2Windows
2. Uninstall previous installation
3. Install new version 2.5.0
4. Log in

### Anything else?

https://github.com/user-attachments/assets/8c187445-c320-4e96-a14e-df319952bd56

**Comment by cryptofyre** [comment]
We're currently unsure as to what is causing this, it seems API related but we cannot pin point it or replicate it.

**Comment by Teeworlds4** [comment]
If you want me to show something or test, that might figure why this is happening, I'm willing to help.

**Comment by cryptofyre** [comment]
So far what we've heard, some folks have solved this by clearing cache with the following:
![Cider_USlc5ur9gp](https://github.com/user-attachments/assets/4c9ee57c-11cf-45af-90dd-5a11f42958b2)

**Comment by Teeworlds4** [comment]
Still doesn't work. Even disabling cache in Ctrl+Shift+I menu and then clearing doesn't work

**Comment by Teeworlds4** [comment]
Patch 2 fixed it
